addons/stock_picking_batch/views/stock_picking_batch_views.xml

```diff
@@ -218,14 +218,25 @@
         <field name="context">{'search_default_draft': True, 'search_default_in_progress': True}</field>
         <field name="search_view_id" ref="stock_picking_batch_filter"/>
         <field name="help" type="html">
-          <p class="o_view_nocontent_smiling_face">
-            Create a new batch transfer
-          </p><p>
-            The goal of the batch transfer is to group operations that may
-            (needs to) be done together in order to increase their efficiency.
-            It may also be useful to assign jobs (one person = one batch) or
-            help the timing management of operations (tasks to be done at 1pm).
-          </p>
+            <div class="container mt-5">
+                <div class="row g-5">
+                    <div class="col-lg-4 opaque-column">
+                        <img src="/stock_picking_batch/static/shapes/wave-picking.svg" class="img-thumbnail w-100 p-4 mb-4" alt="" loading="lazy" style=""/>
```

Comment:
do we need the `loading="lazy"`/`style=""`?

It's not a big deal, just wondering for code cleanliness purposes